sys/src/9/port/devregress.c

```diff
@@ -126,6 +129,12 @@ regresswrite(Chan *c, void *a, i32 n, i64 offset)
 			error("Only v or V");
 		return n;
 
+	case Qwalk:
+		p = a;
+		addr = strtoull(p, 0, 0);
```

Comment:
It may be useful to validate that this is actually a kernel address here. Something like,

    if (addr < KZERO) error("Invalid kernel address");

Reply:
I thought about that but as part of regression, we might want to, rather, let it try to walk it and hope it fails?

Reply:
Ok; sounds fine to me.